R/fit_SE.R

```diff
@@ -100,17 +101,27 @@ fit_FUN <- function(x,
       x <- x[x[[conc]] != 0, ]
     }
     
-    fit_df <- S4Vectors::DataFrame(gDRutils::fit_curves(
-      data.table::as.data.table(x),
-      series_identifiers = nested_identifiers,
-      e_0 = 1,
-      GR_0 = 1,
-      n_point_cutoff = n_point_cutoff,
-      range_conc = range_conc,
-      force_fit = force_fit,
-      pcutoff = pcutoff,
-      cap = cap,
-      normalization_type = curve_type))
+    fit_df <- S4Vectors::DataFrame(
+      gDRutils::fit_curves(
+        data.table::as.data.table(x),
+        series_identifiers = nested_identifiers,
+        e_0 = 1,
+        GR_0 = 1,
+        n_point_cutoff = n_point_cutoff,
+        range_conc = range_conc,
+        force_fit = force_fit,
+        pcutoff = pcutoff,
+        cap = cap,
+        normalization_type = curve_type)
+    )
+    
+    if (is.null(fit_df$fit_source)) {
```

Comment:
Is it possible? How often does it occur? This function adds this https://github.com/gdrplatform/gDRutils/blob/master/R/fit_curves.R#L142

Reply:
This is for safety reasons - if the impossible happens 😉